arrow-core-data/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -1067,7 +1067,11 @@ sealed class Either<out A, out B> : EitherOf<A, B> {
    * The left side of the disjoint union, as opposed to the [Right] side.
    */
   @Suppress("DataClassPrivateConstructor")
-  data class Left<out A> @PublishedApi internal constructor(val a: A) : Either<A, Nothing>() {
+  data class Left<out A> @PublishedApi internal constructor(
```

Comment:
Would making the constructor internal break user code at call sites like `Left(1)`?
Can we do instead something like?:
```kotlin
data class Left<out A> @PublishedApi internal constructor(@Deprecated("use value") val a: A, val value : A = a)
```

Reply:
Nevermind, just realized it was like that before

Reply:
The constructor is already internal. I'm not sure what the difference would be 🤔